src/parthenon_manager.cpp

```diff
@@ -240,17 +240,10 @@ void ParthenonManager::RestartPackages(Mesh &rm, RestartReader &resfile) {
             << std::endl;
 
   const auto file_output_format_ver = resfile.GetOutputFormatVersion();
-  if (file_output_format_ver == -1) {
+  if (file_output_format_ver < HDF5::OUTPUT_VERSION_FORMAT - 1) {
```

Comment:
As above wrt to implied version numbers here.

Reply:
Also added more verbose error message.